content.js

```diff
@@ -134,7 +138,7 @@ function messageListener(request, sender, sendResponse) {
 
     if (request.message == "getVideoID") {
       sendResponse({
-        videoID: getYouTubeVideoID(document.URL)
+        videoID: videoID
```

Comment:
Can you init videoID at the top. `var videoID = false`

Reply:
sure